packages/loader/driver-utils/src/summaryForCreateNew.ts

```diff
@@ -51,6 +51,8 @@ export function isCombinedAppAndProtocolSummary(
  * @param appSummary - Summary of the app.
  * @param protocolSummary - Summary of the protocol.
  * @internal
+ *
+ * @deprecated 2.0.0-internal.3.4.0 - Not intended for public use.  Will be moved to container-loader and no longer exported in an upcoming release.
```

Comment:
my concern with this change as that we lose locality in the code with the related type and type guard. probably fine, but having the code together makes it easier to understand.

Reply:
In the longer-term, the type and type guard should also move and be hidden, just can't happen yet due to the drivers peeking in.

Reply:
i don't know if we'll ever get the drivers that isolated. potentially something like this could move up to protocol in the future, but either way its quite a bit of work to untangle.